Update google-ads.md (#73)
amplitude/amplitude-dev-center: docs/data/sources/google-ads.md

- [Google Ads Customer ID](https://support.google.com/google-ads/answer/1704344?hl=en) of the ad account you want to connect to.
- If you don't have direct access to the account, `Google Ads Manager ID` that you authorized access on which can view this ad account.

### Considerations

- This source only imports **metrics data** from Google Ads. Hence, you can not import other types of data (e.g., experiments).
- Amplitude's Google Ads integration imports data from Google Ads **once a day**, so you will always see your Daily Ad Metrics populated at a certain time. You can check the time of the next sync in the settings.

> - The advertising data you import will be associated with the new Amplitude event called `Daily Ad Metrics` with several event properties, such as `Ad Impressions`, `Ad Clicks`, `Ad Group ID`, etc. For more information, see [Ad-Network Integrations in Amplitude](https://amplitude.com/blog/ad-network-integration)

Author: What if I'm not listing all the items and want to indicate that there are more associated with this event?

## Common issues

### Insufficient permissions

Amplitude's Google Ads Import integration requires that your Google Ads Manager account has administrator privileges. This level of permission allows Amplitude to add and remove users from specific user lists in Google Ads. 
For more information, see [About access levels in your Google Ads Account](https://support.google.com/google-ads/answer/9978556) in Google's documentation. 

### Import job ingests no data 

1. Check if you reject unplanned events in your [Schema settings](https://help.amplitude.com/hc/en-us/articles/360055495852-Configure-the-Schema-settings-to-handle-unexpected-data). If you reject unplanned data,  Amplitude doesn't store the event or its properties.
2. Do your users have corresponding accounts in Amplitude and Google Ads? Google Ads import tries to match users between platforms based on the key-value pairs you selected. If your import job doesn't find any corresponding values, it fails.

### Daily ad metric discrepancies 

Your metrics may occasionally update days after a click occurs. This can happen for many  reasons, including but not limited to:

> - When a conversion occurs days after the initial click

Reviewer: ```suggestion

- When a conversion occurs days after the initial click
```

- When the source detects and removes invalid traffic

> Hence, if Google updates the data afterward - the import job will have to run again to get the updated numbers.

Author: ```suggestion
Hence, if Google updates the data afterward - the import job needs to run again to get the updated numbers.
```

![Google Historical Backfill](../../assets/images/marketing-analytics/google-past-data.png)

For more information on how you can use the data from this integration in Amplitude, see [this blog post](https://amplitude.com/blog/ad-network-integration).

## Common issues

### Insufficient permissions

Amplitude's Google Ads Import integration requires that your Google Ads Manager account has administrator privileges. This level of permission allows Amplitude to add and remove users from specific user lists in Google Ads. 
For more information, see [About access levels in your Google Ads Account](https://support.google.com/google-ads/answer/9978556) in Google's documentation. 

> ### Import job ingests no data 

Reviewer: I think we can remove the numbered list here, and either:

- Use a bulleted list and write an introductory sentence
- Just write the content in paragraph form

Author: I think a bulleted list would work well here. 
I am just not sure how to fix it without a suggestion raised on this (so I can accept a suggestion after you edit it)...

Author: done

> 1. Check if you reject unplanned events in your [Schema settings](https://help.amplitude.com/hc/en-us/articles/360055495852-Configure-the-Schema-settings-to-handle-unexpected-data). If you reject unplanned data - Amplitude will not store the event or its properties.

Reviewer: ```suggestion
1. Check if you reject unplanned events in your [Schema settings](https://help.amplitude.com/hc/en-us/articles/360055495852-Configure-the-Schema-settings-to-handle-unexpected-data). If you reject unplanned data,  Amplitude doesn't store the event or its properties.
```

> You can learn more about handling unplanned event types in this article.

Reviewer: Which article?

Author: Oops! Actually, it references the same article as mentioned in the prev comment (https://help.amplitude.com/hc/en-us/articles/360055495852-Configure-the-Schema-settings-to-handle-unexpected-data), so I think I can just remove this sentence in general

> You can learn more about handling unplanned event types in this article.

Author: ```suggestion
```

> 2. Do your users have corresponding accounts in Amplitude and Google Ads? Google Ads import will try to match users between platforms based on the key-value pairs you selected, so your import job will fail if it does not find any corresponding values.

Reviewer: ```suggestion
2. Do your users have corresponding accounts in Amplitude and Google Ads? Google Ads import tries to match users between platforms based on the key-value pairs you selected. If your import job doesn't find any corresponding values, it fails.
```

> ### Daily Ad Metric discrepancies 

Reviewer: ```suggestion
### Daily ad metric discrepancies 
```

> Your metrics may occasionally update days after a click occurs. This can happen for a number of reasons, including but not limited to:

Reviewer: ```suggestion
Your metrics may occasionally update days after a click occurs. This can happen for many  reasons, including but not limited to:
```

- When a conversion occurs days after the initial click
- When the source detects and removes invalid traffic

> Hence, if Google updates the data afterward - the import job will have to run again to get the updated numbers.

Reviewer: Here, do you mean that if:

1. A user clicks a button
2. Data syncs to Google Ads
3. The user converts

The conversion won't show in Google Ads until the next sync?

Author: Right, a user clicks something -> Amplitude imports data from GA (so we have _click_, but no conversion yet) -> user then converts sometime later: we will see a conversion for this user in GA but not in Amplitude yet, because Amp imports this conversion data during the following sync
